src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx

```diff
@@ -198,7 +198,6 @@ const DictionaryDetailCard = (props) => {
         handleChange={handleChange}
         inputLength={inputLength}
       />
-
```

Comment:
Is this a formatting change?

Reply:
Yeah, it's a formatting change.

Reply:
Can this be done in a different PR please?

Reply:
+1 on this